### DISCUSSION TITLE
How to access the Device Manager?

### DISCUSSION THREAD

**Discussion by paulburd** [responsively-org/responsively-app#1181] [Q&A] [answered]
I'm using the Apple Silicon version (v1.10) and I don't see any way of accessing the Device Manager, or adding/changing the default devices (iPhone 12 Pro, iPad, MacBook Pro). I've seen screenshots here that show a sidebar on the left, but I have no sidebar. What am I missing? Thanks!

<img width="2652" alt="SCR-20240109-pvma" src="https://github.com/responsively-org/responsively-app/assets/1427151/ba898b7f-564d-4acb-b759-e3754dce586b">

**Comment by manojVivek** [comment, marked as answer]
There is no sidebar anymore, the device manager can be accessed here:

![image](https://github.com/responsively-org/responsively-app/assets/1283424/59b67413-a17f-44ad-b826-cc44a0e933b6)

**Reply by paulburd** [reply]
Thank you so much! That icon is so similar to Rotate that I looked right past it. I appreciate the tip. ;)
